vestapol/web_resources/json_resource.py

```diff
@@ -45,21 +49,28 @@ def write_data(self, api_data, destination):
     def unnest_data(self, data, destination):
         pass
 
-    def write_list(self, data: List[dict], destination, data_path=None):
+    def write_list(
+        self,
+        data: List[Dict],
+        destination: DestinationTypes,
+        data_path: str = None,
```

Comment:
`data_path` is a list of tuples.

```suggestion
        data_path: List[Tuple[str,  str]] = None,
```

Reply:
looks like data_path is expected to be a string when passed to `get_pathname`